src/v2/Apps/Fair/Components/FairTiming.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react"
+import { Sans, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { FairTiming_fair } from "v2/__generated__/FairTiming_fair.graphql"
+import { WithCurrentTime } from "v2/Components/WithCurrentTime"
+import { DateTime, Duration } from "luxon"
+
+interface FairMetaProps {
+  fair: FairTiming_fair
+}
+
+const SEPARATOR = <>&nbsp;:&nbsp;</>
+
+const FairTiming: React.FC<FairMetaProps> = ({
+  fair: { exhibitionPeriod, startAt, endAt },
+}) => {
+  return (
+    <>
+      <Text variant="text" color="black80">
```

Comment:
`black80` isn't a valid color

Reply:
I think it is? https://github.com/artsy/palette/blob/2d54be8ea5e8024da86c5c2f948b031c5247025c/packages/palette/src/Theme.tsx#L83

Reply:
Ah interesting: https://palette.artsy.net/tokens/color/